tests/unit/lms/authentication/_helpers_test.py

```diff
@@ -2,24 +2,24 @@
 
 from lms.authentication._helpers import authenticated_userid, get_lti_user, groupfinder
 from lms.validation import ValidationError
-from lms.values import LTIUser
+from tests import factories
 
 
 class TestAuthenticatedUserID:
     @pytest.mark.parametrize(
         "lti_user,expected_userid",
         [
             (
-                LTIUser(
-                    "sam", "Hypothesisf301584250a2dece14f021ab8424018a", "TEST_ROLES"
+                factories.LTIUser(
+                    user_id="sam",
+                    oauth_consumer_key="Hypothesisf301584250a2dece14f021ab8424018a",
```

Comment:
This is a good example of how the factory forces tests to include relevant information but not include noise. These tests do depend on the particular `user_id` and `oauth_consumer_key` values but don't care about `roles`